ml-agents/mlagents/trainers/models.py

```diff
@@ -15,6 +14,19 @@
 EPSILON = 1e-7
 
 
+class Tensor3DShape(NamedTuple):
+    height: int
+    width: int
+    num_channels: int
+
+    @property
+    def gray_scale(self) -> bool:
```

Comment:
is_gray_scale ?

Reply:
I don't think this is actually used. Would you be opposed to removing it (as it doesn't make sense for a tensor to be `gray`)?

Reply:
```
if unused:
    remove()
```